### PR TITLE
[ja] Update content/ja/docs/concepts/overview/components.md file

### PR DESCRIPTION
**Pull request by ziyi-xie**
There are several updates in [the original English text](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/overview/components.md) and therefore this page is outdated.
This PR updated Japanese translation for the [/content/ja/docs/concepts/overview/components.md](https://kubernetes.io/ja/docs/concepts/overview/components/) file.

Fixes https://github.com/kubernetes/website/issues/38544

### CODE REVIEW

**Comment by ziyi-xie** [comment]
/assign @nasa9084
/cc @atoato88 @t-inu
Hi! I've updated Japanese translation for the [/content/ja/docs/concepts/overview/components.md](https://kubernetes.io/ja/docs/concepts/overview/components/) file, would you mind checking if everything looks alright at your convenience time please? Here is [the original English text](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/overview/components.md) and thank you so much!

File: content/ja/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -49,24 +45,25 @@ Kubernetesをデプロイすると、クラスターが展開されます。
...
 
 cloud-controller-managerは、クラウドプロバイダー固有のコントローラーのみを実行します。
-KubernetesをオンプレミスあるいはPC内での学習環境で動かす際には、クラスターにcloud container managerはありません。
+Kubernetesをオンプレミスあるいは個人のPC内での学習環境で動かす際には、クラスターにcloud container managerはありません。
 
-kube-controller-managerを使用すると、cloud-controller-managerは複数の論理的に独立したコントロールループをシングルバイナリにまとめ、これが一つのプロセスとして動作します。パフォーマンスを向上させるあるいは障害に耐えるために水平方向にスケールする(一つ以上のコピーを動かす)ことができます。
+kube-controller-managerと同様に、cloud-controller-managerは複数の論理的に独立したコントロールループをシングルバイナリにまとめ、一つのプロセスとして動作します。パフォーマンスを向上させるあるいは障害に耐えるために水平方向にスケールする(一つ以上のコピーを動かす)ことができます。
 
-次のコントローラーには、クラウドプロバイダーへの依存関係を持つ可能性があります。
+次のコントローラーには、クラウドプロバイダーへの依存関係を持つことができます。
```
Comment (ziyi-xie) [inline content/ja/docs/concepts/overview/components.md:61]: The following is the original text:
> The following controllers can have cloud provider dependencies:

I'm a little confused about the translation of  'dependencies' over here, is '依存関係' the right word?
I'd appreciate it if you could give me some advice about that and thank you so much!

**Comment by atoato88** [comment]
The title of this PR says "Update /content/ja/docs/concepts/services-networking/ingress.md", but the file updated is "content/ja/docs/concepts/overview/components.md".
Is this right title?

**Comment by ziyi-xie** [comment]
/retitle [ja] Update content/ja/docs/concepts/overview/components.md file

**Comment by ziyi-xie** [comment]
@atoato88 I just modified the title and thank you so much for your comment!

File: content/ja/docs/concepts/overview/components.md (hunk 1 above)
Comment (atoato88) [reply content/ja/docs/concepts/overview/components.md:61]:
```suggestion
次のコントローラーは、クラウドプロバイダーへの依存関係を持つことがあります。
```
I think the cloud-controller-manager depends on the cloud services provided by target Cloud Services(e.g. GCE, Azure, AWS).
But the implementation of these defferent on each other, I guess.
So `ことがあります` or `可能性があります` as translation of `can` is better.

`依存関係` is `それぞれのクラウドが提供するサービスに依存している`in precise meaning, I guess.
But it's a too rough free translation, I think `依存関係` is also valid for Ja translation.

File: content/ja/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -105,11 +102,11 @@ Kubernetesによって開始されたコンテナは、DNS検索にこのDNSサ
...
 ### コンテナリソース監視
 
-[コンテナリソース監視](/docs/tasks/debug-application-cluster/resource-usage-monitoring/)は、コンテナに関する一般的な時系列メトリックを中央データベースに記録します。また、そのデータを閲覧するためのUIを提供します。
+[コンテナリソース監視](/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/)は、コンテナに関する一般的な時系列メトリックを中央データベースに記録します。また、そのデータを閲覧するためのUIを提供します。
 
-### クラスターレベルログ
+### クラスターレベルのロギング
 
-[クラスターレベルログ](/docs/concepts/cluster-administration/logging/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
+[クラスターレベルのロギング](/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
```
Comment (atoato88) [inline content/ja/docs/concepts/overview/components.md:109]: EN content has the URL "https://kubernetes.io/docs/concepts/cluster-administration/logging/" as link.
Why do you update this URL with "debug-cluster"?
```suggestion
[クラスターレベルのロギング](/ja/docs/concepts/cluster-administration/logging/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
```

**Review by atoato88 — COMMENTED** [review]
Please check the comments.

File: content/ja/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -49,24 +45,25 @@ Kubernetesをデプロイすると、クラスターが展開されます。
 コントローラーには以下が含まれます。
 
   * ノードコントローラー：ノードがダウンした場合の通知と対応を担当します。
-  * レプリケーションコントローラー：システム内の全レプリケーションコントローラーオブジェクトについて、Podの数を正しく保つ役割を持ちます。
-  * エンドポイントコントローラー：エンドポイントオブジェクトを注入します(つまり、ServiceとPodを紐付けます)。
-  * サービスアカウントとトークンコントローラー：新規の名前空間に対して、デフォルトアカウントとAPIアクセストークンを作成します。
+  * Jobコントローラー：単発タスクを表すJobオブジェクトを監視し、そのタスクを完了まで実行するPodを作成します。
+  * エンドポイントスライスコントローラー：エンドポイントスライスオブジェクトを注入します(つまり、ServiceとPodを紐付けます)。
```
[the remark this answers is not in the document]
Comment (ziyi-xie) [reply content/ja/docs/concepts/overview/components.md:49]: Also confused about this part... 
I'd appreciate it if you could give me some advice about that and thank you so much!

[the remark this answers is not in the document]
Comment (t-inu) [reply content/ja/docs/concepts/overview/components.md:49]: The following page describes the EndPointSlice controller.
https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#management

> Most often, the control plane (specifically, the endpoint slice controller) creates and manages EndpointSlice objects.

So, for easier understanding,  I think "populate" can be translated as "作成する".

[the remark this answers is not in the document]
Comment (ziyi-xie) [reply content/ja/docs/concepts/overview/components.md:49]: @atoato88 @t-inu Thank you so much for your kind instruction!

File: content/ja/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -49,24 +45,25 @@ Kubernetesをデプロイすると、クラスターが展開されます。
 コントローラーには以下が含まれます。
 
   * ノードコントローラー：ノードがダウンした場合の通知と対応を担当します。
-  * レプリケーションコントローラー：システム内の全レプリケーションコントローラーオブジェクトについて、Podの数を正しく保つ役割を持ちます。
-  * エンドポイントコントローラー：エンドポイントオブジェクトを注入します(つまり、ServiceとPodを紐付けます)。
-  * サービスアカウントとトークンコントローラー：新規の名前空間に対して、デフォルトアカウントとAPIアクセストークンを作成します。
+  * Jobコントローラー：単発タスクを表すJobオブジェクトを監視し、そのタスクを実行して完了させるためのPodを作成します。
+  * EndpointSliceコントローラー：EndpointSliceオブジェクトを注入します(つまり、ServiceとPodを紐付けます)。
```
Comment (ziyi-xie) [inline content/ja/docs/concepts/overview/components.md:49]: modified the translation of 'populate'

Comment (ziyi-xie) [inline content/ja/docs/concepts/overview/components.md:49]:
```suggestion
  * EndpointSliceコントローラー：EndpointSliceオブジェクトを作成します(つまり、ServiceとPodを紐付けます)。
```

modified the translation of 'populate'.

**Pull request merged**